cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/store/DatasetBasedTimeScheduleStore.java

```diff
@@ -170,16 +170,18 @@ public void apply() throws Exception {
     }
   }
 
-  private void executePersist(final TriggerKey triggerKey, final Trigger.TriggerState state) {
+  private void executePersist(final TriggerKey triggerKey, final Trigger.TriggerState newTriggerState) {
```

Comment:
Since we are already refactoring. This executePersist() methods saves a changes of state, whereas the other saves a new or changed definition of a job and trigger. Why not name them accordingly: persistChangeOfState() and persistJobAndTrigger()